e2e/benchmarks/benchmark_util.js

```diff
@@ -294,7 +303,11 @@ async function downloadValuesFromTensorContainer(tensorContainer) {
     // Start value downloads from all tensors.
```

Comment:
The patch is not valid for all e2e models. If `tensorContainer` is 'object', the reading may been completed in the other place, take `AutoML image` model for example,  reading the result is https://github.com/tensorflow/tfjs/blob/c513dc30a5cd1820d6c05b4f42ec322b5a038517/tfjs-automl/src/img_classification.ts#L50
Any comments how to handle this issue.